tests/fixtures/build-quiet-cli/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM busybox:latest
```

Comment:
I don't think a new fixture is necessary here - just reuse `build-args` or `build-ctx`